src/Microsoft.VisualStudio.SDK.Analyzers/Utils.cs

```diff
@@ -95,6 +98,39 @@ internal static bool IsDerivedFrom(ITypeSymbol type, ITypeSymbol baseType)
             return false;
         }
 
+        /// <summary>
+        /// Produces the syntax necessary to qualify a simple name.
+        /// </summary>
+        /// <param name="qualifiers">The qualifiers (e.g. the namespace that qualifies a type).</param>
+        /// <param name="simpleName">The simple type name.</param>
+        /// <returns>The qualified type name.</returns>
+        internal static NameSyntax QualifyName(IReadOnlyList<string> qualifiers, SimpleNameSyntax simpleName)
+        {
+            if (qualifiers == null)
+            {
+                throw new ArgumentNullException(nameof(qualifiers));
+            }
+
+            if (simpleName == null)
+            {
+                throw new ArgumentNullException(nameof(simpleName));
+            }
+
+            if (qualifiers.Count == 0)
+            {
+                throw new ArgumentException("At least one qualifier required.");
```

Comment:
Should pass in nameof(qualifiers) as the second arg to specify the argument causing the issue

Reply:
fixed